src/cryptoadvance/specter/templates/device/new_device/new_device_keys.jinja

```diff
@@ -459,6 +459,73 @@
                 if(result==null){
                     return;
                 }
+
+                // foundation's passport returns coldcard format encoded in bcur2
```

Comment:
If passport uses the same format as coldcard it's better to create a function and reuse it instead of copy-pasting the same code.